infra/Makefile

```diff
@@ -3,7 +3,7 @@ environment = dev
 aws_profile = apim-dev
 tf_opt=
 tf_state=-backend-config="key=gpconnectInfra"
-tf_vars=
+tf_vars=-var="environment=$(environment)"
```

Comment:
environment should come from workspace. For infra it's fixed to dev but it's dictated by workspace, i.e. use shouldn't provide a var